[guetzli] Build error on x64-osx

Package: guetzli[core]:x64-osx -> 2020-09-14#1
**Host Environment**
- Host: arm64-osx
- Compiler: AppleClang 13.0.0.13000029
-    vcpkg-tool version: 2999-12-31-unknownhash-debug
    vcpkg-scripts version: 4fff1e9cf 2022-01-17 (34 minutes ago)


**To Reproduce**
`vcpkg install `


**Failure logs**
```
-- Using cached google-guetzli-214f2bb42abf5a577c079d00add5d6cc470620d3.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/google-guetzli-214f2bb42abf5a577c079d00add5d6cc470620d3.tar.gz
-- Applying patch butteraugli.patch
-- Using source at /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean
-- Found external ninja('1.10.2').
-- Configuring x64-osx-dbg
-- Configuring x64-osx-rel
-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:157 (message):
    Command failed: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/downloads/tools/cmake-3.21.1-osx/cmake-3.21.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/x64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:106 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:44 (vcpkg_build_cmake)
  ports/guetzli/portfile.cmake:20 (vcpkg_install_cmake)
  scripts/ports.cmake:145 (include)



```

c++11 must be enabled in the port
<details><summary>/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/install-x64-osx-dbg-out.log</summary>

```
[1/24] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/. -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/guetzli_lib.dir/guetzli/gamma_correct.cc.o -MF CMakeFiles/guetzli_lib.dir/guetzli/gamma_correct.cc.o.d -o CMakeFiles/guetzli_lib.dir/guetzli/gamma_correct.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/gamma_correct.cc
[2/24] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/. -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/guetzli_lib.dir/guetzli/entropy_encode.cc.o -MF CMakeFiles/guetzli_lib.dir/guetzli/entropy_encode.cc.o.d -o CMakeFiles/guetzli_lib.dir/guetzli/entropy_encode.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/entropy_encode.cc
[3/24] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/. -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/guetzli_lib.dir/guetzli/dct_double.cc.o -MF CMakeFiles/guetzli_lib.dir/guetzli/dct_double.cc.o.d -o CMakeFiles/guetzli_lib.dir/guetzli/dct_double.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/dct_double.cc
[4/24] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/. -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_decoder.cc.o -MF CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_decoder.cc.o.d -o CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_decoder.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/jpeg_data_decoder.cc
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/jpeg_data_decoder.cc:28:31: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
...
Skipped 362 lines
...

/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:42:16: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
using ImageF = Image<float>;
               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:146:10: error: unknown type name 'constexpr'
  static constexpr size_t kPointerSize = sizeof(void *);
         ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:146:26: error: expected ';' at end of declaration list
  static constexpr size_t kPointerSize = sizeof(void *);
                         ^
                         ;
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:147:10: error: unknown type name 'constexpr'
  static constexpr size_t kCacheLineSize = 64;
         ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:147:26: error: expected ';' at end of declaration list
  static constexpr size_t kCacheLineSize = 64;
                         ^
                         ;
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:150:31: error: unknown type name 'size_t'
  static void *Allocate(const size_t bytes);
                              ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:155:32: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
using CacheAlignedUniquePtrT = std::unique_ptr<T[], void (*)(void *)>;
                               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:157:31: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
using CacheAlignedUniquePtr = CacheAlignedUniquePtrT<uint8_t>;
                              ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:159:20: warning: default template arguments for a function template are a C++11 extension [-Wc++11-extensions]
template <typename T = uint8_t>
                   ^   ~~~~~~~
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:217:13: warning: alias declarations are a C++11 extension [-Wc++11-extensions]
  using T = ComponentType;
            ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:249:15: warning: rvalue references are a C++11 extension [-Wc++11-extensions]
  Image(Image &&other)
              ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:256:26: warning: rvalue references are a C++11 extension [-Wc++11-extensions]
  Image &operator=(Image &&other) {
                         ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:204:40: error: no member named 'kCacheLineSize' in 'butteraugli::CacheAligned'
    const size_t align = CacheAligned::kCacheLineSize;
                         ~~~~~~~~~~~~~~^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:308:33: error: no member named 'kCacheLineSize' in 'butteraugli::CacheAligned'
    static_assert(CacheAligned::kCacheLineSize % sizeof(T) == 0,
                  ~~~~~~~~~~~~~~^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk/usr/include/c++/v1/__config:896:44: note: expanded from macro 'static_assert'
# define static_assert(...) _Static_assert(__VA_ARGS__)
                                           ^~~~~~~~~~~
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/processor.cc:24:
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:22:
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:326:34: error: a space is required between consecutive right angle brackets (use '> >')
static inline std::vector<Image<T>> CreatePlanes(const size_t xsize,
                                 ^~
                                 > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:329:22: error: a space is required between consecutive right angle brackets (use '> >')
  std::vector<Image<T>> planes;
                     ^~
                     > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:349:34: error: a space is required between consecutive right angle brackets (use '> >')
static inline std::vector<Image<T>> CopyPlanes(
                                 ^~
                                 > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:350:30: error: a space is required between consecutive right angle brackets (use '> >')
    const std::vector<Image<T>> &planes) {
                             ^~
                             > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:351:22: error: a space is required between consecutive right angle brackets (use '> >')
  std::vector<Image<T>> copy;
                     ^~
                     > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:353:30: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (const Image<T> &plane : planes) {
                             ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:392:34: error: a space is required between consecutive right angle brackets (use '> >')
static inline std::vector<Image<T>> PlanesFromPacked(
                                 ^~
                                 > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:394:36: error: a space is required between consecutive right angle brackets (use '> >')
    const std::vector<std::vector<T>> &packed) {
                                   ^~
                                   > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:395:22: error: a space is required between consecutive right angle brackets (use '> >')
  std::vector<Image<T>> planes;
                     ^~
                     > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:397:32: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (const std::vector<T> &p : packed) {
                               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:405:40: error: a space is required between consecutive right angle brackets (use '> >')
static inline std::vector<std::vector<T>> PackedFromPlanes(
                                       ^~
                                       > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:406:30: error: a space is required between consecutive right angle brackets (use '> >')
    const std::vector<Image<T>> &planes) {
                             ^~
                             > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:409:28: error: a space is required between consecutive right angle brackets (use '> >')
  std::vector<std::vector<T>> packed;
                           ^~
                           > >
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include/butteraugli/butteraugli.h:411:30: warning: range-based for loop is a C++11 extension [-Wc++11-extensions]
  for (const Image<T> &image : planes) {
                             ^
In file included from /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/processor.cc:24:
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:30:1: error: unknown type name 'constexpr'
constexpr int kButteraugliStep = 3;
^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:38:40: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  void Compare(const OutputImage& img) override;
                                       ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:40:32: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  void StartBlockComparisons() override;
                               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:41:33: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  void FinishBlockComparisons() override;
                                ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:44:48: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
                   int factor_x, int factor_y) override;
                                               ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:47:51: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
                      int off_x, int off_y) const override;
                                                  ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:49:42: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  double ScoreOutputSize(int size) const override;
                                         ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:51:44: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  bool DistanceOK(double target_mul) const override {
                                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:55:44: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  const std::vector<float> distmap() const override { return distmap_; }
                                           ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:56:35: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  float distmap_aggregate() const override { return distance_; }
                                  ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:58:33: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
  float BlockErrorLimit() const override;
                                ^
/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/./guetzli/butteraugli_comparator.h:63:41: warning: 'override' keyword is a C++11 extension [-Wc++11-extensions]
      std::vector<float>* block_weight) override;
                                        ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
31 warnings and 20 errors generated.
[18/24] /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++  -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/. -I/Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg_installed/x64-osx/include -fPIC -g -arch x86_64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.1.sdk -MD -MT CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_reader.cc.o -MF CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_reader.cc.o.d -o CMakeFiles/guetzli_lib.dir/guetzli/jpeg_data_reader.cc.o -c /Users/leanderSchulten/git_projekte/Lichtsteuerung/vcpkg/buildtrees/guetzli/src/cc470620d3-8a0908bd04.clean/guetzli/jpeg_data_reader.cc
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**
<details><summary>vcpkg.json</summary>

```
{
  "name": "lichtsteuerung",
  "version-string": "2.0.0",
  "dependencies": [
    "guetzli"
  ]
}

```
</details>

--- a/ports/guetzli/CMakeLists.txt
+++ b/ports/guetzli/CMakeLists.txt
@@ -17,6 +17,8 @@ list(REMOVE_ITEM guetzli_srcs "guetzli/guetzli.cc")
 
 add_library(guetzli_lib ${guetzli_srcs})
 
+target_compile_features(guetzli_lib PUBLIC cxx_std_11)
+
 add_executable(guetzli guetzli/guetzli.cc)
 
 target_link_libraries(guetzli_lib ${butteraugli_library} ${PNG_LIBRARIES})

--- a/ports/guetzli/portfile.cmake
+++ b/ports/guetzli/portfile.cmake
@@ -9,17 +9,16 @@ vcpkg_from_github(
     PATCHES butteraugli.patch
 )
 
-file(COPY ${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt DESTINATION ${SOURCE_PATH})
+file(COPY "${CMAKE_CURRENT_LIST_DIR}/CMakeLists.txt" DESTINATION "${SOURCE_PATH}")
 
-vcpkg_configure_cmake(
-    SOURCE_PATH ${SOURCE_PATH}
-    PREFER_NINJA
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS_DEBUG -DDISABLE_INSTALL_HEADERS=ON -DDISABLE_INSTALL_TOOLS=ON
 )
 
-vcpkg_install_cmake()
+vcpkg_cmake_install()
 vcpkg_copy_pdbs()
 
-vcpkg_copy_tool_dependencies(${CURRENT_PACKAGES_DIR}/tools/guetzli)
+vcpkg_copy_tool_dependencies("${CURRENT_PACKAGES_DIR}/tools/guetzli")
 
-file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/guetzli RENAME copyright)
+file(INSTALL "${SOURCE_PATH}/LICENSE" DESTINATION "${CURRENT_PACKAGES_DIR}/share/guetzli" RENAME copyright)

--- a/ports/guetzli/vcpkg.json
+++ b/ports/guetzli/vcpkg.json
@@ -1,11 +1,15 @@
 {
   "name": "guetzli",
-  "version-string": "2020-09-14",
-  "port-version": 1,
+  "version-date": "2020-09-14",
+  "port-version": 2,
   "description": "Perceptual JPEG encoder",
   "homepage": "https://github.com/google/guetzli",
   "dependencies": [
     "butteraugli",
-    "libpng"
+    "libpng",
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    }
   ]
 }

--- a/scripts/ci.baseline.txt
+++ b/scripts/ci.baseline.txt
@@ -408,7 +408,6 @@ gstreamer:x64-osx=fail
 gtk:x64-windows-static=fail
 gtk:x64-windows-static-md=fail
 gts:x64-osx=fail
-guetzli:x64-osx=fail
 halide:x64-windows-static=fail
 hdf5:arm64-windows=fail
 hdf5:arm-uwp=fail

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2630,7 +2630,7 @@
     },
     "guetzli": {
       "baseline": "2020-09-14",
-      "port-version": 1
+      "port-version": 2
     },
     "guilite": {
       "baseline": "2021-07-31",

--- a/versions/g-/guetzli.json
+++ b/versions/g-/guetzli.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a7da0a5fa0b5bdb568edcbde81b314c3f7089cb0",
+      "version-date": "2020-09-14",
+      "port-version": 2
+    },
+    {
       "git-tree": "2adfb6dcd5eab92b258f45aaf4a494bed18eb687",
       "version-string": "2020-09-14",
       "port-version": 1

